pkg/aws/client.go

```diff
@@ -484,6 +509,19 @@ func (c *awsClient) CheckAdminUserNotExisting(userName string) (err error) {
 	return nil
 }
 
+func (c *awsClient) CheckAdminUserExists(userName string) (err error) {
+	userList, err := c.iamClient.ListUsers(&iam.ListUsersInput{})
+	if err != nil {
+		return err
+	}
+	for _, user := range userList.Users {
+		if *user.UserName == userName {
+			return nil
+		}
+	}
+	return fmt.Errorf("IAM user '%s' does not exist. Run `rosa init` first", userName)
```

Comment:
This should be `return false, nil`, and that error message should be part of `cmd/create/cluster/cmd.go`